src/libs/ajax/Billing.ts

```diff
@@ -59,6 +59,15 @@ export interface GCPBillingProject extends BaseBillingProject {
   servicePerimeter?: string;
 }
 
+export const isAzureBillingProject = (project?: BillingProject): project is AzureBillingProject =>
```

Comment:
The types for billing projects live in this file, so adding the utility methods here as well. At some point types and utility methods should probably move out of the ajax file.